templates/nvidia-guardrails-with-RAG/README.md

```diff
@@ -0,0 +1,130 @@
+# Guardrails Integration with LangChain Expression Language Template
+
+This example demonstrates integrating NeMo Guardrails with LangChain Expression Language Templates. Here we start of with a `nvidia-rag-canonical` from the LangChain [templates](https://github.com/langchain-ai/langchain/tree/master/templates/nvidia-rag-canonical) which performs RAG using Milvus Vector Store and NVIDIA Models, both embedding and Chat.
```

Comment:
If people use this, they'll be starting from this template `nvidia-guardrails-with-rag` instead

Reply:
Yes. Got it! Apologies for the typo. Let me know how I can make the necessary changes